### PR TITLE
Fe news8

### PR DESCRIPTION
**Pull request by ConnorK519**

### CODE REVIEW

File: src/components/CreateComment.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            placeholder="Enter your comment here..."
+            className="commentInput"
+            value={newComment}
+            onChange={(event) => {
+              setNewComment(event.target.value);
+            }}
+          ></textarea>
+        </label>
+        <button onClick={handleSubmit}>Post</button>
+      </form>
```
Comment (Jodiech) [inline src/components/CreateComment.jsx:50]: Currently, I can click the post button multiple times, is there a way you can add some validation to stop a user from posting multiple blank comments?

File: src/components/CreateComment.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComment(keyId, newComment).then(({ createdComment }) => {
+      setComments((currentComments) => {
+        setNewComment("");
+        return [createdComment, ...currentComments];
+      });
+    });
+  };
```
Comment (Jodiech) [inline src/components/CreateComment.jsx:27]: Don't forget to add some error handling here 👀

File: src/components/CreateComment.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <>
+      <form className="commentInputBox">
+        <label>
+          Post a comment here:
+          <textarea
```
Comment (Jodiech) [inline src/components/CreateComment.jsx:35]: Nice use of textarea here 👍

**Review by Jodiech — CHANGES_REQUESTED** [review]
Hey 👋 nice work on this ticket! There are a few comments dotted around for you to take a look at. Please implement something to prevent a user from posting multiple times! Please resubmit when you're ready and give us a shout if you need anything 👍

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Connor, very nice work on this one! 

Please make sure that the 'Post' button in disabled for the time of posting so it can't be clicked multiple times for the same comment and won't post it many times. 
Also a message that the comment is being posted would be nice too. (In case of a slow connection.)

Please try to implement these, then you can merge your code. 👍

**Pull request merged**